k2/python/k2/fsa_algo.py

```diff
@@ -571,3 +575,10 @@ def invert(fsa: Fsa) -> Fsa:
         ragged_arc, aux_labels, _ = _k2.invert(fsa.arcs, fsa.aux_labels,
                                                need_arc_map)
         return Fsa(ragged_arc, aux_labels)
+
+
+def create_sparse(row_indexes: torch.Tensor, col_indexes: torch.Tensor,
```

Comment:
Can this one be removed as there is a similar version in utils.py that is with more detailed doc?

Reply:
Sure, remove it pls.

Reply:
BTW we can have those extra args in both the pruned and un-pruned version of dense intersection.